lib/osutils/file.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Do we need tests for file backup, remove tasks etc?

Reply:
Ok I will add some